packages/shared/src/api/DashboardVisibility.ts

```diff
@@ -0,0 +1,6 @@
+enum DashboardVisibility {
+  PUBLIC = 'PUBLIC',
```

Comment:
there is no need to to set the vaule exactly like the name. PUBLIC, PRIVATE should be enough